pkg/services/sqlstore/migrations/folder_mig.go

```diff
@@ -35,6 +35,10 @@ func addFolderMigrations(mg *migrator.Migrator) {
 		Type: migrator.UniqueIndex,
 		Cols: []string{"title", "parent_uid", "org_id"},
 	}))
+
+	mg.AddMigration("Test", migrator.NewRawSQLMigration("").Postgres(`
+		CREATE TABLE fts(title TEXT, ts tsvector)
+	`))
 }
```

Comment:
This is a temporary migration to make sure integration tests of Grafana pass with `tsvector` is present in table schemas. I will remove it once I get some 👀 on this PR.